eox_nelp/tests/test_principal_views.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test views file.
+"""
+
+from django.test import TestCase
+from django.urls import reverse
+
+import eox_nelp
+
+
+class EOXInfoTestCase(TestCase):
```

Comment:
my concern is do we need two different tests here ? I mean there is a reason  to create two different tests when we can test both cases in just one method and the result  is always the same , or the version won't appear if `ALLOWED_HOST` is the default value

Reply:
yes but maybe could be a case where eox-info URL is working respond 200 but in a different version.
or is improbably? xD

Reply:
I think that view has not changed in the last  3 years in another plugins

Reply:
yes, I prefer that test case, just one last comment, the name should be just `test_views.py` generally this follow the same structure of the container folder.

Reply:
yes but, this are only views of the first level folder of the plugin. Dont test all views of the plugin. Is ok to use `test_views.py`?

Reply:
thast file should just test the views at the first level, if there are more views files those could be tested inside the parent folder or you can add a folder with the same name inside the test folder.

so if you want to test course_api/views, you can create a test folder inside course_api, and then create another test_views.py file 

or you can create a course_api inside current test folder and the the test_views file

Reply:
knowing that test_views will be referred to views of each module or folder, I agree.
[201584b](https://github.com/eduNEXT/eox-nelp/pull/26/commits/201584b4f0848f3bf53a3926ba4d180c5aedba6f)